packages/ui/toggletip/index.jsx

```diff
@@ -27,7 +29,7 @@ export class Toggletip extends Component {
       context: {
         contentRef: this.setContentRef,
         triggerRef: this.setTriggerRef,
-        events: { onClick: this.onToggle },
+        events: { onClick: this.onToggle, onMouse: this.onToggle },
```

Comment:
@ailincaibianca Does this enable the toggletip to open/close on a mouse over event ?
If so, I believe we should control this through a prop, as the default behaviour up until now was without these mouse events.

Reply:
fixed